cfgov/unprocessed/apps/owning-a-home/css/explore-rates.less

```diff
@@ -2,6 +2,13 @@
    "Rate checker" custom styles
    ========================================================================== */
 
+// Mixin to remove any default spacing or borders on an element.
+.u-reset {
```

Comment:
I think we should define this as `.u-reset() {` so it doesn't get included in the output css